<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1040</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonAddNode">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add Node</string>
    </property>
   </widget>
   <widget class="QLabel" name="dataLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>1101</width>
      <height>421</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="NodeNumAddEdit" name="addNumberText">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>140</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDeleteNode">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Delete Node</string>
    </property>
   </widget>
   <widget class="NodeNumDeleteEdit" name="deleteNumberText">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>180</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QRadioButton" name="deleteMultipleRadioButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>190</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Delete all matches</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="indexSpinBox">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>150</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="indexLabel">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>150</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Index:</string>
    </property>
   </widget>
   <widget class="QLabel" name="indexMessageLabel">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>60</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0 = Head, can inster up to max length</string>
    </property>
   </widget>
   <widget class="pictureLabel" name="linkedListHeaderCodeLabel">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>70</y>
      <width>569</width>
      <height>603</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>NodeNumAddEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>NodeNumAddEdit.h</header>
  </customwidget>
  <customwidget>
   <class>NodeNumDeleteEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>NodeNumDeleteEdit.h</header>
  </customwidget>
  <customwidget>
   <class>pictureLabel</class>
   <extends>QLabel</extends>
   <header location="global">pictureLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
